ergo-lib/src/wallet/tx_builder.rs

```diff
@@ -332,7 +326,7 @@ mod tests {
     #[test]
     fn test_empty_inputs() {
```

Comment:
I guess this test should be restructured since tx builder won't be returning EmptyInputBoxSelection error anymore but it will fail right away when trying to convert empty vec into bounded vec

Reply:
Actually, this test does not make sense in tx builder anymore. Neither in BoxSelection. It's a BoxSelector test now, when there are not enough assets to select but I think we already have that covered.